### ISSUE TITLE
mom locales option error in Devuan Chimaera

### ISSUE THREAD

**Issue by charliemartinez** [pieroproietti/penguins-eggs-legacy#119]
**Reproduce:**

1. eggs mom
2. tools  clean/locales/skel/yolk
3. locales install/clean locales

**Result:**
```
remove package locales
creating /etc/default.locale
       Error: ENOENT: no such file or directory, open
      '/usr/lib/penguins-eggs/conf/distros/chimaera/locales/locale.template'
      Code: ENOENT
Press enter to continue
```

**Comment by pieroproietti** [comment]
Hi Charlie

You are right, I removed locales.template and was thinking to remove the command ```eggs tools locales``` too. 
Originally. I thought to something can be usefull for internationalization, but after due the fact who calamares take all inside I forgot about this. 
That command can be usefull too, installing with krill so, perhaps in the future I will put it in order again. 
At the moment thanks for segnalation and waiting for inspiraction: that you think? I think it is convenient to  remove it and implement locale configuration inside krill installer.

I have similar problem with the command update too: here we have the option ```basket```, a way to get eggs new releases  from sourceforge via a service running on the host penguins-eggs.net. Actually using a more common penguins-eggs-ppa there is not reason to exist too.

**Comment by pieroproietti** [comment]
I rewrote the command: ```eggs tools locales``` and I think to solved the problem, both in the command and in krill where I added the possibility to set all the languages.

I just tested on Debian bullseye, and probably next days I will touch it a bit more, in all the way I tryied it against es_AR and keyboard latam and it happen to work nicelly.

Just releasing eggs-9.0.45.

Thanx for suggestion Charlie
